src/Math-Complex/PMComplex.class.st

```diff
@@ -198,6 +198,13 @@ PMComplex >> abs [
 	^ ((real * real) + (imaginary * imaginary)) sqrt
 ]
 
+{ #category : #arithmetic }
+PMComplex >> abs2 [
```

Comment:
Consider renaming this to `squareModulus` or `absSquared`.

Reply:
ok I will rename it as absSquared.